lib/perl/Genome/Qc/Tool/VerifyBamId.pm

```diff
@@ -7,6 +7,16 @@ use List::MoreUtils qw(any);
 
 class Genome::Qc::Tool::VerifyBamId {
     is => 'Genome::Qc::Tool::WithVariationListVcf',
+    has => [
+        default_genotype_vcf_file_id => {
+            is => 'Text',
+            is_optional => 1,
+        },
+        default_genotype_vcf_file => {
+            is => 'Genome::SoftwareResult::ImportedFile',
```

Comment:
Would it makes sense for this to be a `Genome::SoftwareResult::StageableSimple::SingleFile` so it could be either an imported file or one of our Genotype VCFs?  It doesn't seem likely as a use case, but should we rule it out?